src/screens/BlockUser/BlockUser.tsx

```
@@ -75,81 +75,177 @@ const Requests = (): JSX.Element => {
+    async (user: InterfaceUserPg): Promise<void> => {
       try {
+        const userId = user.id;
         const { data } = await blockUser({
           variables: {
             userId,
-            orgId: currentUrl,
+            organizationId: currentUrl,
           },
         });
-        if (data) {
+        if (data && data.blockUser && data.blockUser.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)